conf/docker/solr/.gitignore

```diff
@@ -0,0 +1,2 @@
+solr-7.3.0.tgz
```

Comment:
Is this intended? I noticed below that we are no longer downloading solr with wget so should it be included?

Reply:
if somebody wants to build a solr image, they will still need to download solr then paste it in conf/docker/solr